facebookresearch/ParlAI
# 1. Shell and python file to only download image chat data. 2. Updated…

**Patch description**
1. Shell and python file to only download `image_chat` data. 
2. Updated Readme for the same.

The current setup to download just the data is cumbersome requiring full knowledge of ParlAI platform. Making it easy for researchers who just want to use the data and build their code on top of different repos. Basically a direct api for the `build` in `parlai.tasks.image_chat.build`

**Testing steps**
Enter steps to test your pull request. Give a clear and concise description of
what you expected to happen during testing.

Run as `./parlai/tasks/image_chat/download_data.sh` to download only data. (Updated the README.) 

**Logs**
```
Running code from: 
Downloading in data root:
[ downloading: http://parl.ai/downloads/image_chat/image_chat.tgz to .....]
```

**Other information**
@klshuster please verify if this is the correct data.
